### PR TITLE
nodejs-week1

### PR DESCRIPTION
**Pull request by Yuliia-Balandiuk**

### CODE REVIEW

File: src/backend/app.js [hunk 1]
```diff
@@ -19,16 +20,69 @@ app.use(express.json());
...
 app.use(cors());
 
-router.use("/meals", mealsRouter);
+router.use('/meals', mealsRouter);
+
+const errMessage = 'Sorry, there are no meals.';
+
+app.get('/future-meals', async (req, res) => {
+  const [row] = await knex.raw(
+    ' SELECT * FROM `meal` WHERE `when` > "2022-08-11"'
```
Comment (Divyajg) [inline src/backend/app.js]: Instead of hardcoding the date, you can use JavaScript Date methods for now.

File: src/backend/app.js [hunk 2]
```diff
@@ -19,16 +20,69 @@ app.use(express.json());
 
 app.use(cors());
 
-router.use("/meals", mealsRouter);
+router.use('/meals', mealsRouter);
+
+const errMessage = 'Sorry, there are no meals.';
+
+app.get('/future-meals', async (req, res) => {
+  const [row] = await knex.raw(
```
Comment (Divyajg) [inline src/backend/app.js]: Can use just a variable to assign data.

**Review by Divyajg — COMMENTED** [review]
Everything looks fine, but I suggest changing the hard coded date, to mitigate issues in the future.

File: src/backend/app.js (hunk 1 above)
Comment (Yuliia-Balandiuk) [reply src/backend/app.js]: Thank you for your review.
I agree with you and will fix it.

File: src/backend/app.js (hunk 2 above)
Comment (Yuliia-Balandiuk) [reply src/backend/app.js]: Sorry, I don't understand, what you mean. Can you explain, please?

Comment (Divyajg) [reply src/backend/app.js]: > Instead of "[rows]", "rows" is also fine.

**Comment by viedamayanti** [comment]
Good work Julia 💪💪

**Pull request merged**